### ISSUE TITLE
Problem: MS Teams failing with switch to PowerAutomate

### ISSUE THREAD

**Issue by markush81** [ansible-semaphore/semaphore#2358]
### Issue

Microsoft announced to stop Webhooks, instead we should use Powerautomate.

https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/

But Powerautomate sends different return code, so following patch would be needed

```
diff --git a/services/tasks/alert.go b/services/tasks/alert.go
index 509bc47a..345819b0 100644
--- a/services/tasks/alert.go
+++ b/services/tasks/alert.go
@@ -352,7 +352,7 @@ func (t *TaskRunner) sendMicrosoftTeamsAlert() {
 
        if err != nil {
                t.Log("Can't send microsoft teams alert! Error: " + err.Error())
-       } else if resp.StatusCode != 200 {
+       } else if resp.StatusCode != 200 && resp.StatusCode != 202 {
                t.Log("Can't send microsoft teams alert! Response code: " + strconv.Itoa(resp.StatusCode))
        }
```

### Impact

Service (scheduled tasks, alerts)

### Installation method

Package

### Semaphore Version

2.10.22

**Comment by markush81** [comment]
Another point though is that Microsoft doesn't support version 1.5 when sent via Powerautomate https://github.com/MicrosoftDocs/msteams-docs/issues/9801.

So 1.4. would be more compatible.

**Comment by markush81** [comment]
included in v2.10.30
